ppa/archive/util.py

```diff
@@ -32,13 +33,17 @@ class HathiImporter:
     SUCCESS = 1
     #: status - skipped because already in the database
     SKIPPED = 2
+    #: rsync error
+    RSYNC_ERROR = 3
 
     #: human-readable message to display for result status
     status_message = {
         SUCCESS: 'Success',
         SKIPPED: 'Skipped; already in the database',
         hathi.HathiItemNotFound: 'Error loading record; check that id is valid.',
+        # possibly irrelevant with removal of data api code
```

Comment:
It's referenced mostly through `TestHathiBaseAPI` in [test_hathi.py](https://github.com/Princeton-CDH/ppa-django/blob/c8c66d620129fa0dab6301661ba33450a53c750d/ppa/archive/tests/test_hathi.py#L173) Am I right in guessing this PR makes that class unnecessary as well?

Reply:
We still need Hathi API code for the bibliographic API. The comments indicate the 403 is specific to data api, but it seems like it could still occur, so I'm going to leave the exception and error handling in for now.